etna/datasets/utils.py

```diff
@@ -187,6 +187,11 @@ def match_target_quantiles(features: Set[str]) -> Set[str]:
     return {i for i in list(features) if pattern.match(i) is not None}
 
 
+def match_target_components(features: Set[str]) -> Set[str]:
+    """Find target components in dataframe columns."""
```

Comment:
We don't have any columns and dataframes inside this function. May be we should change description a little bit.

Reply:
Fixed